### ISSUE TITLE
Undefined being added to URL parameter value

### ISSUE THREAD

**Issue by wallslide** [iron-meteor/iron-url#15]
My route looks like this:

```javascript
Router.map(function() {
  this.route('ship.parcel.addressChooser', {path: '/ship/choose-address/:_id',});
});
```

And I call Router.go() like this:
```javascript
Router.go('ship.parcel.addressChooser', {_id: parcelId});
```

The parcelId is a string that equals `PP-dev-KrdGbz`

The resulting url that I get sent to ends up being this: `http://localhost:3002/ship/choose-address/undefinedPP-dev-KrdGbz`

You can see that there is an extra `undefined` in that URL. When going deep into my debugger, I found the following:

![image](https://cloud.githubusercontent.com/assets/917069/7929279/e803d946-0936-11e5-99c4-08a3bb6de29b.png)

Basically, the parameter `format` is undefined, which then gets appended as a string to the final path. I'm not sure what the purpose of that match parameter is, but it seems like you could just do the same thing that you are doing with the `slash` match parameter [here](https://github.com/iron-meteor/iron-url/blob/fbed043a15a020a934fa192ea656a31cabd69395/lib/url.js#L323) right below that line like this: `format = format || '';`

**Comment by wallslide** [comment]
Created a pull request with the change. https://github.com/iron-meteor/iron-url/pull/16

**Comment by solace** [comment]
I can reproduce this issue. Traced it to the line indicated at 338 of @wallslide's screenshot.

**Comment by cmather** [comment]
should be fixed in 1.0.9
